package.json

```diff
@@ -12,7 +12,12 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint --no-fix"
+    "build-and-test": "NODE_ENV=test npm run build && npm test",
+    "lint": "vue-cli-service lint --no-fix .",
+    "test": "npm run cypress:run && npm run nyc:report",
+    "cypress:run": "cypress run",
+    "cypress:open": "cypress open",
+    "nyc:report": "nyc report"
```

Comment:
Is this report part of Cypress?

Reply:
here's how it's working:
- `npm run cypress:run` runs tests and collects code coverage
- `npm run nyc:report` generates a code coverage report based on the test runs

I've put it as a part of `npm test` so we'll get code coverage after each run.